pulp_2to3_migration/app/plugin/content.py

```diff
@@ -181,8 +181,8 @@ async def run(self):
                 # We are waiting on the coroutine to finish, because the order of the processed
                 # content for plugins like Container and RPM is important because of the relations
                 # between the content types.
-                await asyncio.wait(
-                    [self.migrate_to_pulp3(pulp_2to3_detail_qs, self.migrator, ctype, pb=pb)]
+                await asyncio.gather(
+                    self.migrate_to_pulp3(pulp_2to3_detail_qs, self.migrator, ctype, pb=pb)
```

Comment:
And actually to be honest I'm not sure the ```gather()``` is necessary either.  This is a non-async for-loop so  ```await```ing here should intrinsically mean that it doesn't move on until the coroutine returns.

Reply:
I agree.
Does it make sense to make the async loop? Thoug the majority of the operations we have here are the db ones, so they are blocking anyway :/

Reply:
I don't think it makes sense to use an async for loop